scripts/qa_crowbarsetup.sh

```diff
@@ -1356,6 +1364,10 @@ EOF
         fi
     fi
 
+    if [[ $hcp = 1 ]]; then
+        add_hcp_repo
```

Comment:
for a single line function maybe we can put the zypper ar here, but I am OK like it is now